app/lib/frontend/email_sender.dart

```diff
@@ -157,7 +158,11 @@ class _GmailSmtpRelay implements EmailSender {
       _accessToken = null;
       throw EmailSenderException.failed();
     } on MailerException catch (e, st) {
```

Comment:
How about just doing:
```dart
} on SmtpNoGreetingException catch (e, st) {
  _logger_warning(...);
```

Printing a slightly different error message is nice too.